app/javascript/onboarding/components/Navigation.jsx

```diff
@@ -42,7 +42,15 @@ class Navigation extends Component {
         >
           {!hidePrev && (
             <button onClick={prev} className="back-button" type="button">
-              Back
+              <svg
+                width="16"
+                height="16"
+                fill="none"
```

Comment:
Hmm will this conflict with the styles? Not sure if onboarding uses theme styles though.

The SVGs I've been working with had a `fill` attribute and was causing issues for me.

Reply:
It looked to me like the crayons style was using the current color from the button to fill in the svg, so I removed the figma-generated fill color 🤷 
<img width="1800" alt="Screen_Shot_2020-04-22_at_4_24_21_PM" src="https://user-images.githubusercontent.com/6921610/80043212-eb4f2100-84b5-11ea-80f3-45cfb25522a1.png">